docs/manpage-urls.json

```
@@ -0,0 +1,5 @@
+{
+"fuzzel.ini(5)": "https://man.archlinux.org/man/fuzzel.ini.5",
+"tofi(5)": "https://github.com/philj56/tofi/blob/master/doc/tofi.5.md",
+"ncmpcpp": "https://manpages.debian.org/testing/ncmpcpp/ncmpcpp.1.en.html"
```

Should it be like this or am I mistaken?

```suggestion
"ncmpcpp(1)": "https://manpages.debian.org/testing/ncmpcpp/ncmpcpp.1.en.html"
```

btw, this file is indented weirdly, is that just JSON or perhaps should the contents of the attrset be indented